File: Results/DER/Translation/translation_output/w_test/codeqwen/codenet/java/python/p03264.py
Here is the Python equivalent of the Java code:

```python
class p03264:
    def main(self):
        n = int(input())
        
        o = 0
        e = 0
        
        for i in range(1, n+1):
            if i % 2 == 1:
                o += 1
            else:
                e += 1
        print(o*e)

p03264().main()
```

This Python code does the same thing as the Java code. It reads an integer from the user, counts the number of odd and even numbers from 1 to the input number, and then prints the product of the counts of odd and even numbers.

File: Results/DER/Translation/translation_output/w_test/codeqwen/codenet/python/java/p03264.java
Here is the Java equivalent of the Python code you provided:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int k = scanner.nextInt();
        System.out.println((int) Math.ceil(k / 2.0) * (k / 2));
    }
}
```

This Java code does the same thing as the Python code. It reads an integer from the user, calculates the ceiling of the integer divided by 2, and then multiplies that result by the integer divided by 2. The result is then printed to the console.
